gems/sorbet-runtime/test/types/builder_syntax.rb

```diff
@@ -438,28 +438,6 @@ def self.foo; end; foo
         end
       end
 
-      it 'forbids empty notify' do
```

Comment:
should this go in pay-server to test our configuration?

Reply:
yep!

> Some tests get deleted because they were specific to the `notify:` parameter. I'll be re-introducing these as tests of our T::Configuration handler in pay-server when I do that migration.